UniversalAlgebra/Algebras/Setoid.lagda

```diff
@@ -46,10 +52,10 @@ First we define an operator that translates an ordinary signature into a signatu
 \begin{code}
 
 ⟦_⟧s : {α ρ : Level} → Signature 𝓞 𝓥 → Setoid α ρ → Setoid _ _
-⟦ 𝑆 ⟧s ξ .Carrier = Σ[ f ∈ ∣ 𝑆 ∣ ] ((∥ 𝑆 ∥ f) → (Carrier ξ))
+⟦ 𝑆 ⟧s ξ .∣_∣ = Σ[ f ∈ (fst 𝑆) ] ((∥ 𝑆 ∥ f) → ∣ ξ ∣)
```

Comment:
To make things look nicer, I often use a local `open Setoid` in a `where` clause; I'll do local renames there too.